pax-macro/src/lib.rs

```diff
@@ -630,7 +630,9 @@ pub fn pax_derive(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
         } else {
             unreachable!()
         };
-        let current_dir = std::env::current_dir().expect("Unable to get current directory");
+        let current_dir = PathBuf::from(
+            std::env::var("CARGO_MANIFEST_DIR").expect("Unable to get current directory"),
+        );
```

Comment:
`current_dir` expects a `src` file that is only available at the root project folder